.npmignore

```diff
@@ -0,0 +1,2 @@
+test
```

Comment:
Why test folder?

Reply:
We don't need users to download test dir when running `npm install`